asv_bench/benchmarks/arithmetic.py

```diff
@@ -469,4 +469,29 @@ def time_apply_index(self, offset):
         offset.apply_index(self.rng)
 
 
+class Sub:
```

Comment:
can you give a more informative name?  e.g. this is specific to subtraction with a MultiIndex

Reply:
Done